tests/test_runner.py

```diff
@@ -1,16 +1,25 @@
 import logging
+import shutil
 import sys
 from pathlib import Path
-from typing import Generator
+from typing import Generator, Tuple
 
 import numpy as np
 import pytest
 
 from pymchelper.input_output import fromfile
 from pymchelper.executor.options import SimulationSettings
-from pymchelper.executor.runner import Runner
+from pymchelper.executor.runner import Runner, SimulatorType
 
 
+@pytest.fixture
+def topas_path() -> Generator[Tuple[Path, Path], None, None]:
```

Comment:
Either name it differently, say:
`topas_mock_and_input` or split into two fixtures.